test/common/access_log/access_log_formatter_fuzz_test.cc

```diff
@@ -10,8 +10,9 @@ namespace {
 
 DEFINE_PROTO_FUZZER(const test::common::access_log::TestCase& input) {
   try {
+    // Remove invalid characters in request/response formatters.
     std::vector<AccessLog::FormatterProviderPtr> formatters =
-        AccessLog::AccessLogFormatParser::parse(input.format());
+        AccessLog::AccessLogFormatParser::parse(replaceInvalidCharacters(input.format()));
```

Comment:
Should we have PGV annotations for this rather than substitution?

Reply:
Hm, I have this PR out specifically for that purpose because I'd like to have a PGV constraints for string to `not_contain` a null character, I bumped it last week, but will do again.
https://github.com/envoyproxy/protoc-gen-validate/pull/253

Actually a lot of time is being spend replacing invalid characters across all fuzzers and it'd be nice to either have this PGV constraint, or specifically guide mutations to not contain nulls.

Reply:
Yeah, maybe remove this for now and then wait until https://github.com/envoyproxy/protoc-gen-validate/pull/253 lands; I agree that the PGV constraint is the right long term solution, not only for performance, but because Envoy will reject bad configs on the wire when we add these annotations to the API.